### DISCUSSION TITLE
Hide routing options for specific profiles

### DISCUSSION THREAD

**Discussion by Jmrcm** [osmandapp/OsmAnd#14327] [Q&A]
# 🚀 feature request

### Description

Hello, I would like some developer to be interested in working on map profiles such as: ship, plane, sky, cavalcade, etc.  For example, on a ship, nothing should appear about traffic alerts, streets, pedestrians, that is, nothing on land, rather it should show depth alerts, docks, piers, ocean routes, lighthouses, beaches, etc.  By plane: airports, runways, landing strips, air corridors, etc. and so on with profiles such as: sky, cavalcade, etc.  The idea is to make these profiles show more real data and alerts according to their characteristics, although it is understood that it is not intended to reach the level of an application specialized in this type of navigation, at least the data displayed in Osmand would be more real giving more professionalism to the application, otherwise it is better that these profiles disappear from the app.  Kind regards, thank you very much.
![Screenshot_2022-05-15-21-27-33-162_net osmand plus~3](https://user-images.githubusercontent.com/83715100/168504993-90e6dbf0-8d29-4a48-9d81-bfee8824c582.jpg)

**Comment by vshcherb** [comment]
You can't hide existing properties but you can create custom routing configuration where you can just delete it. I.e. copy pasting routing profile and change it.
